payments-ui-core/src/main/java/com/stripe/android/ui/core/forms/USBankAccountSpec.kt

```diff
@@ -0,0 +1,12 @@
+package com.stripe.android.ui.core.forms
+
+import androidx.annotation.RestrictTo
+import com.stripe.android.ui.core.elements.LayoutSpec
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+val USBankAccountParamKey: MutableMap<String, Any?> = mutableMapOf(
+    "type" to "us_bank_account"
+)
```

Comment:
@michelleb-stripe since we aren't using `FormViewModel`, does this need to be anything? Can this be an empty map?

Reply:
yeah, this can be an empty map

Reply:
This field is being removed from SupportedPaymentMethods in an upcoming PR, so no worries, you can just leave it.